### ISSUE TITLE
qq: why did you switch to chromadb?

### ISSUE THREAD

**Issue by mikayelh** [kevin-fruitful/chadgpt-backend#1]
@kevin-fruitful hi! just wanted to ask why did you switch away from Deep Lake (full disclosure, I'm from Deep Lake team, trying to understand how to make the package better!)

**Comment by kevin-fruitful** [comment]
Hey there, @mikayelh!  First off, let me just say that I've got nothing but respect for what you all are doing over there. I'm a bit embarrassed to admit this, but the switch had less to do with any shortcomings of Deep Lake and more with a "late-night-coding-induced" mistake on my end.

You see, it was late into the night, my code was a tangle, my eyes were blurry, and my brain was doing gymnastics trying to keep up. In this state of disarray, I forgot one of the basics of Python: proper instantiation of instance variables. Instead of debugging like a seasoned sailor, I kept switching DBs, hoping the problem would magically disappear. Unfortunately, that's akin to changing the entire ship when the real issue was a loose bolt.

Ultimately, it was ChromeDB's logging feature which made me realize my error: I was inadvertently summoning and banishing the database with each initialization, like a sleep-deprived sorcerer. So, the switch was less about escaping from Deep Lake and more about stumbling into enlightenment (and perhaps needing a stronger coffee).

I hope my misadventure brings a smile to your face, and rest assured that in future (well-rested) projects, Deep Lake is still on my radar. Your tool is fantastic, and it was my oversight that led to this comedy of errors. Keep up the great work! 💻☕🌛
